Source/w_wad.c

```diff
@@ -264,10 +264,21 @@ static void W_CoalesceMarkedResource(const char *start_marker,
           is_marked = 0;                          // stop marking lumps
         }
       else
-        if (is_marked)                            // if we are marking lumps,
-          {                                       // move lump to marked list
-            marked[num_marked] = *lump;
-            marked[num_marked++].namespace = namespace;  // killough 4/17/98
+        if (is_marked || lump->namespace == namespace)
```

Comment:
I don't understand the second part of this check.

Reply:
// sf: check for namespace already set

Reply:
You may be right, removing this check makes no difference in my testing.